types/testsuite/suite.go

```diff
@@ -38,16 +38,16 @@ type TestSuite struct {
 
 	Ctx sdk.Context
 
-	AccountKeeper   authkeeper.AccountKeeper
-	AolKeeper       aolkeeper.Keeper
-	AolMsgServer    aoltypes.MsgServer
-	BankKeeper      bankkeeper.Keeper
-	BurnKeeper      burnkeeper.Keeper
-	DIDMsgServer    didtypes.MsgServer
-	DIDKeeper       didkeeper.Keeper
-	TokenKeeper     tokenkeeper.Keeper
-	MarketKeeper    marketkeeper.Keeper
-	MarketMsgServer markettypes.MsgServer
+	AccountKeeper     authkeeper.AccountKeeper
+	AolKeeper         aolkeeper.Keeper
+	AolMsgServer      aoltypes.MsgServer
+	BankKeeper        bankkeeper.Keeper
+	BurnKeeper        burnkeeper.Keeper
+	DIDMsgServer      didtypes.MsgServer
+	DIDKeeper         didkeeper.Keeper
+	TokenKeeper       tokenkeeper.Keeper
+	DatadealKeeper    datadealkeeper.Keeper
+	DatadealMsgServer datadealtypes.MsgServer
```

Comment:
How about modifying it with camel expression here as well?
```suggestion
	DataDealKeeper    datadealkeeper.Keeper
	DataDealMsgServer datadealtypes.MsgServer
```